push-api/JSONParser/yamlParser.py

```diff
@@ -4,11 +4,10 @@ class yamlParser(object):
     def __inti__(self):
```

Comment:
`def __inti__(self):` => `def __init__(self):`

Or remove it altogether, why bother with the variable if it is not used.